wca-regulations.md

```diff
@@ -446,9 +446,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - B4) Blindfolded phase:
     - B4a) The competitor dons the blindfold to start the blindfolded phase.
     - B4b) The competitor must not apply moves to the puzzle before they have fully donned the blindfold. Penalty: disqualification of the attempt (DNF).
-    - B4c) The judge must ensure that there is a sight blocker between the competitor's face and the puzzle while the competitor is solving.
+    - B4c) After the competitor has applied the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.
```

Comment:
A small nit: "after" technically applies to the entire solving period. We can use an alternative that makes it clear that the goal is to cover during as much of the solving period as possible.

```suggestion
    - B4c) As soon as the competitor applies the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.
```